packages/components/tag/src/tag.vue

```diff
@@ -75,6 +75,9 @@ const handleClick = (event: MouseEvent) => {
 
 const handleVNodeMounted = (vnode: VNode) => {
   // @ts-ignore
-  vnode.component.subTree.component.bum = null
+  if (vnode?.component?.subTree?.component?.bum) {
```

Comment:
```suggestion
  if(vnode.component!.subTree.type === 'transition-stub') return
```
I think this will make the intent of the code clearer

Reply:
I don't think we should be test specific but rather protect the code in general (not just for Vue Test Utils but in case some Transition internals change, etc.)